src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/EncodeURL.txt

```diff
@@ -0,0 +1,16 @@
+// ******** STRING PARAMETERS ********
+
+>> EncodeUrl("https://make.powerapps.com/")
+"https%3A%2F%2Fmake.powerapps.com%2F"
+
+>> EncodeUrl("powerfx&test")
+"powerfx%26test"
+
+>> EncodeUrl("")
```

Comment:
>EncodeUrl("") [](http://example.com/codeflow?start=3&length=13)

Please also add test cases where the function receives as argument:
- Blank
- Error
- Unsupported type (number, date, ...).
- Concatenation e.g. EncondeUrl("www" & "msft.com/")
- Concatenation with coercion e.g. EncondeUrl("www.msft.com/?orderid=" & numberVariable)
- Call other functions within e.g. EncondeUrl(Upper("www.msft.com/"))

Also:
- Use EncondeUrl within With function e.g. With({url:"www.msft.com/"}, EncondeUrl(url))

Reply:
Added more tests. Please let me know if there's more scenario that you want to add :)

Reply:
One more, actually:
>> EncodeUrl(Error({Kind:ErrorKind.Div0}))
Error({Kind:ErrorKind.Div0})

Reply:
Updated! (`EncodeUrl(" ")` already exists and `EncodeUrl("[http://b*()!\'ool](http://b%2A%28%29%21%5C%27ool/)")` should expect http%3A%2F%2Fb%2A%28%29%21%5C%27ool